oaknational/Oak-Web-Application
# feat: use keystages MV in subject listing page query [LESQ-1012]

## Description

Music year: 1955

- refactor subject listing page query to use `mv_key_stages` because this gives us just the keystages we need ahead of `all keystages` being published


## How to test

I tested this locally by running OWA pointing at staging, where all keystages is already published. Initially the subject listing page was broken:
<img width="600" alt="Screenshot 2024-08-08 at 08 06 23" src="https://github.com/user-attachments/assets/5251f3c0-953a-48b0-84ce-bc3c19e0b66c">
and after the change in the query the page is working:
<img width="600" alt="Screenshot 2024-08-08 at 08 07 11" src="https://github.com/user-attachments/assets/7cbd0008-77ed-4b9a-a80e-484b1c118e08">
I did this again having published the new keystage locally and refreshing MVs, in case there were any data discrepancies on staging, and had the same results.


1. Go to https://deploy-preview-2676--oak-web-application.netlify.thenational.academy
2. Click on a keystage
3. The subject listing pages should look ✨ the same ✨
